pkgs/ffigen/lib/src/config_provider/config_types.dart

```diff
@@ -420,3 +416,249 @@ class VarArgFunction {
 
   VarArgFunction(this.postfix, this.types);
 }
+
+class PackingValue {
+  int? value;
+  PackingValue(this.value);
+}
+
+class ConfigImpl implements Config {
```

Comment:
Should this go in a separate file? It seems somewhat different from the rest of the code in this file.